# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../vserver/typo_hotfix.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- patch-3.14.17-vs2.3.6.13.diff.orig	2014-08-30 12:13:46.000000000 -0500
+++ patch-3.14.17-vs2.3.6.13.diff	2014-09-09 08:54:05.000000000 -0500
 -7477,7 +7477,7 @@ diff -NurpP --minimal linux-3.14.17/incl
  
  extern kuid_t make_kuid(struct user_namespace *from, uid_t uid);
  extern kgid_t make_kgid(struct user_namespace *from, gid_t gid);
-+extern krag_t make_ktag(struct user_namespace *from, gid_t gid);
++extern ktag_t make_ktag(struct user_namespace *from, vtag_t tag);
  
  extern uid_t from_kuid(struct user_namespace *to, kuid_t uid);
  extern gid_t from_kgid(struct user_namespace *to, kgid_t gid);
